pkg/transforms/outputdata.go

```diff
@@ -47,6 +48,9 @@ func (f OutputData) SetOutputData(edgexcontext *appcontext.Context, params ...in
 		return false, err
 	}
 	// By setting this the data will be posted back to to configured trigger response, i.e. message bus
+	if len(f.ResponseContentType) > 0 {
```

Comment:
Do we have real use case for zero length response content type? If not, I'd suggest to remove it this checking since it is kind of redundant.

Reply:
The zero length response content type use case is default and will set the content-type header to text/plain. We are checking for use of the explicit set of the response type, and then set that in the context to eventually pass off to the response writer.

Reply:
ok, so looks like there is a use case for it then. thanks for clarifying that